odxtools/dataobjectproperty.py

```diff
@@ -174,6 +174,3 @@ def convert_bytes_to_physical(self,
 
     def is_valid_physical_value(self, physical_value):
         return self.compu_method.is_valid_physical_value(physical_value)
-
-    def get_valid_physical_values(self):
```

Comment:
I personally use `get_valid_physical_values`, so it would be nicer to keep it.

Reply:
okay, but maybe it should be moved out of the base class because it is not possible to define this for all DOPs. (what are the valid values of a structure?)

Reply:
It's fine to return an empty list in that case

Reply:
okay, the function is now still gone, but for texttables you can retrieve the same data using the compu-method's `get_scales()` method:
```python
choices = [ s.compu_const for s in my_texttable_compumethod.get_scales() ]
```